tools/metadata_viewer/kvstore.py

```diff
@@ -96,8 +99,11 @@ def decode(self):
 
         ret['key_space'] = self._decode_ks(keyspace)
         ret['key_buf'] = key_buf
-        data_rdr = Reader(self.v_stream)
-        data = data_rdr.read_optional(lambda r: r.read_iobuf())
+        if self.optional_value:
```

Comment:
`optional_value` is a bit misleading in my opinion, how about `value_is_optional_type`?